### ISSUE TITLE
Monk bootshine true strike

### ISSUE THREAD

**Issue by LOTJFLek** [MKhayle/XIVComboPlugin#355]
New monk level 7 the combo isnt working on bootshine, true strike

**Comment by Erumite** [comment]
I don't believe it's currently designed to work that way and completely automate the combo.  MNK combo system isn't like normal melee 1-2-3 combos - later on you'll get alternate weapon skills you can use during different forms.  Using one will buff the other for the next time you get into that form. 

The current setup has it so you're still pressing 1-2-3, but it will automatically swap between the stance actions based on buffs.  eg: Dragon Kick if you don't have the resources for a buffed Bootshine, else Bootshine.
